src/Tribe/Template.php

```diff
@@ -519,19 +533,32 @@ public function template( $name, $context = array(), $echo = true ) {
 		// Setup the Hook name
 		$hook_name = implode( '/', $namespace );
 
-		// Check if the file exists
-		$file = $this->get_template_file( $name );
+		// Cache file location and existence.
+		if ( ! isset( $file_exists[ $cache_name_key ] ) || ! isset( $files[ $cache_name_key ] ) ) {
+			// Check if the file exists
+			$files[ $cache_name_key ] = $file = $this->get_template_file( $name );
 
-		// Check if it's a valid variable
-		if ( ! $file ) {
-			return false;
+			// Check if it's a valid variable
+			if ( ! $file ) {
+				return $file_exists[ $cache_name_key ] = false;
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 1ea55291c738059f736e8f3290e41b72</sub>